src/transformers/models/time_series_transformer/configuration_time_series_transformer.py

```diff
@@ -132,13 +135,13 @@ class TimeSeriesTransformerConfig(PretrainedConfig):
 
     def __init__(
         self,
-        input_size: int = 1,
         prediction_length: Optional[int] = None,
         context_length: Optional[int] = None,
         distribution_output: str = "student_t",
         loss: str = "nll",
+        input_size: int = 1,
         lags_sequence: List[int] = [1, 2, 3, 4, 5, 6, 7],
```

Comment:
Could you add docs for lags_sequence, also stating that they need to provide an empty list in case they don't want lags?